src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Grid, Paper, Divider, Button, Link } from '@material-ui/core';
+import './VideoCard.styles.css';
+
+export default function VideoCard({ video }) {
+  const { id, snippet } = video;
+  return (
+    <Grid item>
+      <Paper className="paper">
+        <img
+          src={snippet.thumbnails.medium.url}
+          alt={snippet.title}
+          className="thumbnail"
+        />
+        <Divider />
+        <div className="paperTitle">
+          <Link href={`/watch/${id.videoId}`} target="_self" rel="noopener noreferrer">
+            {snippet.title}
+          </Link>
```

Comment:
The same comment here: instead of using `href` you should be using `to` so it doesn't make a refresh of the page